src/HonzaBotner/appsettings.json

```diff
@@ -8,8 +8,9 @@
     },
     "AllowedHosts": "*",
     "DiscordRoles": {
-        "RolesMapping": {
-            ""
+        "RoleMapping": {
+            "B-18000-SUMA-STUDENT": 0,
+            "B-18100-SUMA-ZAMESTNANEC": 0,
```

Comment:
Přidat:

ICT = B-18301-ZAMESTNANEC
PR = B-18913-ZAMESTNANEC
TEACHER = B-181** ... not sure if there is B-18100-ZAMESTNANEC or something (B-18100 is katedry)

Reply:
Context: https://github.com/fit-ctu-discord/honza-botner-dev/issues/2

Reply:
Potřebujeme studenta?